lib/helper/WebDriverIO.js

```diff
@@ -668,6 +670,26 @@ class WebDriverIO extends Helper {
   }
 
   /**
+   * Return text from random element, if there is several elements with the same locator
+   */
+  grabTextFromRandomElement(locator) {
```

Comment:
no random elements, please. 

you can get the number of visible elements, so use Math.random() inside user code to get the element.
I'd propose to pull out this method from the core as well as grabAttributeFromRandomElement.
We should avoid recommending to use something randomly in tests.

**Remove this method**